kiva/fonttools/tests/test_font.py

```diff
@@ -30,17 +30,19 @@ def test_find_font_empty_name(self):
         # the path from which the font manager loads font files, then this test
         # can be less fragile.
         font = Font(face_name="")
-        font_file_path = font.findfont()
+        font_file_path, face_index = font.findfont()
```

Comment:
This is a breaking change to a public API. I believe there exists downstream code that calls findfont like this.
Is this intended? Alternative to breaking the API, one can add a method for returning the index and keeping `findfont` unchanged?

Reply:
> I believe there exists downstream code that calls findfont like this.

Or maybe not. Not sure.

Reply:
There _shouldn't_ be downstream code which calls this, but that doesn't mean there _isn't_.

Reply:
I see. In that case, if this signature is to be changed, perhaps it could be changed to return an object that wraps the file path and the index?
If we really really need to restore the old behaviour (e.g. it breaks things downstream), it might be enough to implement `__fspath__ ` to make the new object pathlike.

Reply:
Done